types/emmetio__implicit-tag/tslint.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-single-declare-module": false,
+        "npm-naming": [
+            true,
+            {
+                "mode": "code",
+                "errors": [
+                    [
+                        "NoDefaultExport",
+                        false
+                    ],
+                    [
+                        "NeedsExportEquals",
+                        false
+                    ]
+                ]
+            }
+        ]
+    }
```

Comment:
It looks like the lint errors are correct. The file that the `"main"` property in package.json points to is a CommonJS file that uses `module.exports = ...`.

```suggestion
```

Reply:
Implemented your suggestions